paper-fab.html

```diff
@@ -103,12 +108,14 @@
       iron-icon {
         @apply(--paper-fab-iron-icon);
       }
-      
+
       span {
         width: 100%;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        text-align: center;
+        @apply(--paper-fab-label);
```

Comment:
Is there a good reason why we need to expose a mixin here? Seems a little too powerful.